hack/mlflow/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+trap 'kill $(jobs -p)' EXIT
+
+kubectl exec  -n postgres deploy/postgres -- psql -h localhost -U ps_user --no-password -p 5432 -d ps_db -c "CREATE DATABASE mlflow;"
+
+export AWS_ACCESS_KEY_ID=llm-operator-key
+export AWS_SECRET_ACCESS_KEY=llm-operator-secret
+
+kubectl port-forward -n minio service/minio 9000 9090 &
+sleep 1
+
+bucket_name=mlflow
+aws --endpoint-url http://localhost:9000 s3 mb s3://${bucket_name}
+
+# There is no official Helm chart for MLflow, so we can use the community chart or bitnamicharts/mlflow.
+# The community chart hits a bug (https://github.com/community-charts/helm-charts/issues/46), so we use
+# bitnamicharts/mlflow here.
+#
+# See https://github.com/mlflow/mlflow/issues/6118 and https://github.com/bitnami/charts/tree/main/bitnami/mlflow.
+
+helm upgrade --install --create-namespace -n mlflow mlflow oci://registry-1.docker.io/bitnamicharts/mlflow
+
+https://github.com/mlflow/mlflow/blob/master/examples/deployments/deployments_server/openai/example.py
```

Comment:
not sure, need to run this example?

Reply:
Ah sorry. just forgot to remove. 

This was an example for deploying a gateway for OpenAI. Wanted to experiment later.
```suggestion
```